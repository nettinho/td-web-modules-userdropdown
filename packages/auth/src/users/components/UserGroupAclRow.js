import React from "react";
import PropTypes from "prop-types";
import { withRouter } from "react-router-dom";
import { Table } from "semantic-ui-react";

export const UserGroupAclRow = ({ resource, role, group }) => (
  <Table.Row>
    <Table.Cell content={resource} />
    <Table.Cell content={group} />
    <Table.Cell content={role} />
  </Table.Row>
);

UserGroupAclRow.propTypes = {
  resource: PropTypes.string,
  role: PropTypes.string,
  group: PropTypes.string
};

export default withRouter(UserGroupAclRow);
