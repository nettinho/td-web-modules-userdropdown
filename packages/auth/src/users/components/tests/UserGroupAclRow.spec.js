import React from "react";
import { shallow } from "enzyme";
import { UserGroupAclRow } from "../UserGroupAclRow";

describe("<UserGroupAclRow />", () => {
  const acl = {
    resource: "resource1",
    role: "role1",
    group: "grupo1"
  };

  const props = { acl };

  it("matches the latest snapshot", () => {
    const wrapper = shallow(<UserGroupAclRow {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
