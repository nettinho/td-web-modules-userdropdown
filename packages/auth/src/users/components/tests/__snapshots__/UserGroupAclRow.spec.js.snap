// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserGroupAclRow /> matches the latest snapshot 1`] = `
<TableRow
  as="tr"
  cellAs="td"
>
  <TableCell
    as="td"
  />
  <TableCell
    as="td"
  />
  <TableCell
    as="td"
  />
</TableRow>
`;
